pkg/resources/pods/additional_labels.go

```diff
@@ -1,34 +1,43 @@
 package pods
 
-var labelsFor7_9 map[string]string = map[string]string{
-	"com.company":   "Red_Hat",
-	"rht.prod_name": "Red_Hat_Integration",
-	"rht.prod_ver":  "2022.Q1",
-	"rht.comp":      "Broker_AMQ",
-	"rht.comp_ver":  "7.9.2",
-	"rht.subcomp":   "broker-amq",
-	"rht.subcomp_t": "application",
-}
+const (
+	DEFAULT_COMP_VER = "default_comp_ver"
+)
 
-var labelsFor7_10 map[string]string = map[string]string{
+var labelsForRh map[string]string = map[string]string{
 	"com.company":   "Red_Hat",
 	"rht.prod_name": "Red_Hat_Integration",
-	"rht.prod_ver":  "2022.Q2",
 	"rht.comp":      "Broker_AMQ",
-	"rht.comp_ver":  "7.10.0",
 	"rht.subcomp":   "broker-amq",
 	"rht.subcomp_t": "application",
 }
 
-var labelsFromVersion map[string]map[string]string = map[string]map[string]string{
-	"7.9.2":  labelsFor7_9,
-	"7.10.1": labelsFor7_10,
+var productVerFromImageVer map[string]string = map[string]string{
+	"7.9.1":  "2021.Q4",
+	"7.9.2":  "2022.Q1",
+	"7.9.3":  "2022.Q1",
+	"7.9.4":  "2022.Q2",
+	"7.10.0": "2022.Q2",
+	"7.10.1": "2022.Q3",
+
+	DEFAULT_COMP_VER: "2022.Q3",
```

Comment:
@brusdev is his default comp ver the best choice? As it is it gives us a window to work with into the future. But really we don't want to be having a default.

Reply:
@gtully I guess DEFAULT_COMP_VER would be another value to update but I see your point it is better a wrong value that an unknown value
2022.Q3 is already out, see https://access.redhat.com/documentation/en-us/red_hat_integration/2022.q3
should we predict 2022.Q4?

Reply:
I changed my mind maybe we should use a past value, i.e. `2021.Q4`.
Ideally, DEFAULT_COMP_VER should be used only for versions not present in the productVerFromImageVer maps, i.e. 7.9.0, 7.8.7 ...
Moreover using a past value we don't need to update it

Reply:
I agree, it is a bit strange to have any sense of a default, but it can mean in the past, as we will have to update this in the future.
we can let the default be a historical line in the sand. 2021.Q4 seems sensible.
the other option is to not set it or to use "unknown" -  but that may break any parser expecting a quarter date.